tests/python/frontend/pytorch/test_forward.py

```diff
@@ -0,0 +1,134 @@
+    input_shape = [1, 3, 224, 224]
+    input_data = torch.rand(input_shape).float() * 256
+    imagenet_mean = [123., 117., 104.]
+    imagenet_stdev = [58.395, 57.12, 57.375]
+    for channel in range(3):
+        input_data[:, channel] -= imagenet_mean[channel]
+        input_data[:, channel] /= imagenet_stdev[channel]
+    return model, input_data
+
+
+def load_mnist():
+    from mnist import Net
+    model = Net()
+    input_shape = [1, 1, 28, 28]
+    input_data = torch.rand(input_shape).float() * 256
+    return model, input_data
+
+
+def load_model(model_name):
+    if hasattr(torchvision.models, model_name):
+        return load_torchvision(model_name)
+    elif hasattr(pretrainedmodels, model_name):
+        return load_pretrainedmodels(model_name)
+    elif model_name.startswith('mobilenet'):
+        return load_mobilenet(model_name)
+    elif model_name == 'mnist':
+        return load_mnist()
+    else:
+        raise RuntimeError('Model not supported')
+
+
+def verify_model(model_name):
+    baseline_model, baseline_input = load_model(model_name)
+    baseline_output = baseline_model(baseline_input).detach().numpy()
+    dtype = 'float32'
+    input_name = 'input0'
+    input_shapes = {input_name: list(baseline_input.shape)}
+    output_shape = baseline_output.shape
+    ctx = tvm.cpu()
+    trace = torch.jit.trace(baseline_model, baseline_input)
+    print(trace.graph)
+    with TemporaryDirectory() as tmp:
+        path = os.path.join(tmp, 'model.pth')
+        torch.jit.save(trace, path)
+        print('SAVED MODEL')
+        sym, params = nnvm.frontend.from_pytorch(path, input_shapes)
+    print('SUCCESSFULLY GOT SYM')
+    for k, v in params.items():
+        print(f'{k} = {v.dtype}{v.shape}')
+    compiled_input = {input_name: tvm.nd.array(baseline_input.numpy())}
+    graph, lib, params = nnvm.compiler.build(sym, 'llvm', input_shapes,
+#                                             dtype=dtype_dict,
+                                             dtype='float32',
+                                             params=params)
+#    nnvm.compiler.build_module._remove_noref_params(params, graph)
+    compiled_model = tvm.contrib.graph_runtime.create(graph, lib, ctx)
+    compiled_model.set_input(**params)
+    compiled_model.set_input(**compiled_input)
+    compiled_model.run()
+    compiled_output = compiled_model.get_output(
+        0, tvm.nd.array(np.zeros(output_shape).astype(dtype), ctx)).asnumpy()
+    tvm.testing.assert_allclose(baseline_output, compiled_output, rtol=1e-4, atol=1e-4)
+
+
+model_names = pretrainedmodels.model_names
+model_names.extend(['mobilenet_v1', 'mobilenet_v2', 'mnist'])
+
+
+model_name = 'vgg11_bn'
+model_name = 'bninception'
+model_name = 'dpn107'
+model_name = 'nasnetalarge'
+model_name = 'fbresnet152'
+model_name = 'inceptionresnetv2'
+model_name = 'xception'
+model_name = 'senet154'
+
+if __name__ == '__main__':
+
+#    for model_name in model_names:
+    for model_name in [model_name]:
```

Comment:
why not model_names?

Reply:
Thanks, yeah these are debugging artifacts. I plan to have a separate list `supported_models` that will be traversed in the actual test since not all models are supported yet.